src/app/services/editors/gameobject/gameobject-loot-template.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+
+import { GameobjectHandlerService } from '../../handlers/gameobject-handler.service';
+import { QueryService } from '../../query.service';
+import {
+  GAMEOBJECT_LOOT_TEMPLATE_TABLE,
+  GameobjectLootTemplate
+} from '../../../types/gameobject-loot-template.type';
+import { LootEditorService } from '../loot-editor.service';
+import {
+  GAMEOBJECT_TEMPLATE_ID, GAMEOBJECT_TEMPLATE_LOOT_ID,
+  GAMEOBJECT_TEMPLATE_TABLE
+} from '../../../types/gameobject-template.type';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GameobjectLootTemplateService extends LootEditorService<GameobjectLootTemplate> {
```

Comment:
the extra logic added in `LootEditorService` is only handling the part when a loot ID must be assigned in the main template table of the entity. For example `creature_loot_template` is linked to `lootId` of `creature_template`, skinning/pickpocketing loot also have their linked IDs.

Does `gameobject_loot_template` have a loot id in `gameobject_template` ? If not, it should just extend `MultiRowEditorService` instead

Reply:
I'll solve this later.
I will consider this:
```
Only gameobject type 3 (GAMEOBJECT_TYPE_CHEST) or 25 (GAMEOBJECT_TYPE_FISHINGHOLE) use data1 as loot ID, for other types data1 is used in other ways
```
[[source]](http://www.azerothcore.org/wiki/loot_template#loot_template-Entry)